pkg/handlers/helper.go

```diff
@@ -139,48 +137,46 @@ func alertName(a template.Alert) (*string, error) {
 	return nil, fmt.Errorf("no alertname defined in alert")
 }
 
-// SendServiceLog sends a servicelog notification for the given alert
```

Comment:
Also, a style suggestion - could we even move the whole method into `handlers/service_log`? Just to make it more organised.

Reply:
@hectorakemp - That's a good point. Actually, it's my intention to keep the `ServiceLogHandler` focusing solely on API communication. The `SendServiceLog` function, on the other hand, will handle the business logic, such as forming the appropriate summary and description based on the firing status.

As we expand the service log handler to include additional OCM API endpoints in the future, I expect that it continues to specialize in API interactions without delving into the business logic. WDYH?

Reply:
Good point Shawn - I'm pretty new to OA - so I'm not sure what the overall architectural intentions of the operator are in the future. I personally don't mind either way. WDYT @bmeng ?

Reply:
I have no preference here, but seems this kind of make the `helper` itself a bit ambiguous.
It looks to me move the service log related functions to the `service_logs.go` looks more clear

Reply:
> I have no preference here, but seems this kind of make the `helper` itself a bit ambiguous. It looks to me move the service log related functions to the `service_logs.go` looks more clear

I was thinking to keep a clear distinction between API communication and business logic to make `SendServiceLog()` separate from the `handlers/service_logs.go`. But your suggestion to have service log related functions in one place also makes sense. I will update accordingly then.